index.html

```diff
@@ -81,6 +81,209 @@ <h1 class="heading-primary">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+          <!-- Work 1 -->
+          <div class="work">
+            <div class="work-image-holder"></div>
+            <div class="work-details">
+              <h3 class="heading-tertiary">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="work-features">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+          <!-- Work 1 -->
+          <div class="work">
+            <div class="work-image-holder"></div>
+            <div class="work-details">
+              <h3 class="heading-tertiary">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="work-features">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!----------------------->
+      <!-- ABOUT ME SECTION -->
+      <!----------------------->
+
+      <section class="section-about-me">
+        <div class="about-me-header">
+          <h2 class="heading-secondary">About me</h2>
+          <p class="about-me-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <a href="#" class="btn">Get my resume</a>
+          <img
+            class="about-me-top-bg"
+            src="imgs/about-me-top.png"
+            alt="Beautiful illustration"
+          />
+          <img
+            class="about-me-bottom-bg"
+            src="imgs/about-me-bottom .png"
+            alt="Beautiful illustration"
+          />
+        </div>
+
+        <!-- CARDS -->
+        <div class="cards">
+          <!-- Languages card -->
+          <div class="card">
+            <img
+              src="imgs/card-icons/icon-languages.png"
+              alt=""
+              class="card-icon"
+            />
+            <h3 class="heading-tertiary">Languages</h3>
```

Comment:
- [ ] Kindly put the technology categories ("Languages", "Frameworks" and "Skills") in a ul, each of them in a li